docs/warnings.rst

```diff
@@ -27,7 +27,7 @@ script using the `warnings.filterwarnings` function as follows::
      >>> import warnings
      >>> from astropy.io import fits
      >>> warnings.filterwarnings('ignore', category=UserWarning, append=True)
-     >>> fits.writeto(filename, data, clobber=True)
```

Comment:
I didn't check but this example probably predates the clobber/overwrite change, and is here to show how to filter warnings typically from an invalid FITS file. So replacing clobber by overwrite is fine for me (or you can also remove this argument).